data/mods/Magiclysm/monsters/monsters.json

```diff
@@ -242,5 +242,92 @@
     "death_function": [ "NORMAL" ],
     "special_attacks": [ { "type": "bite", "cooldown": 30 }, [ "GRAB", 15 ], [ "scratch", 15 ], [ "slam", 10 ], [ "SMASH", 30 ] ],
     "flags": [ "SEES", "SMELLS", "KEENNOSE", "PATH_AVOID_DANGER_1", "WARM", "GRABS", "BLEED", "REGENERATES_1", "FLAMMABLE" ]
+  },
+  {
+    "id": "mon_stirge",
+    "type": "MONSTER",
+    "name": "stirge",
+    "description": "This horrid flying creature looks like a cross between a large bat and oversized mosquito.",
+    "default_faction": "magical_beast",
+    "bodytype": "flying insect",
+    "species": [ "MAGICAL_BEAST" ],
+    "diff": 2,
+    "volume": "1000 ml",
+    "weight": 1250,
+    "hp": 20,
+    "speed": 133,
+    "material": [ "iflesh" ],
+    "symbol": "s",
+    "color": "red",
+    "aggression": 10,
+    "morale": 10,
+    "melee_skill": 4,
+    "melee_dice": 1,
+    "melee_dice_sides": 4,
+    "melee_cut": 2,
+    "dodge": 4,
+    "vision_day": 20,
+    "vision_night": 20,
+    "harvest": "stirge_death",
+    "anger_triggers": [ "PLAYER_CLOSE" ],
+    "death_function": [ "NORMAL" ],
+    "flags": [ "SEES", "SMELLS", "HEARS", "FLIES", "HIT_AND_RUN", "PATH_AVOID_FIRE", "SWARMS" ]
+  },
+  {
+    "id": "mon_shrieker",
+    "type": "MONSTER",
+    "name": "shrikier",
+    "description": "A shrieker is a human-sized mushroom that emits a piercing screech to drive off creatures that disturb it.",
+    "default_faction": "magical_beast",
+    "species": [ "FUNGUS" ],
+    "diff": 2,
+    "volume": "62500 ml",
+    "weight": 81500,
+    "hp": 25,
+    "speed": 100,
+    "material": [ "veggy" ],
+    "symbol": "7",
+    "color": "light_gray",
+    "aggression": 0,
+    "morale": 0,
+    "melee_cut": 0,
+    "dodge": 0,
+    "vision_day": 10,
+    "vision_night": 10,
+    "harvest": "exempt",
+    "special_attacks": [ [ "SHRIEK_ALERT", 1 ] ],
+    "anger_triggers": [ "PLAYER_CLOSE" ],
+    "death_function": [ "NORMAL" ],
+    "flags": [ "SEES", "NOHEAD", "POISON", "IMMOBILE", "NO_BREATHE", "PACIFIST" ]
+  },
+  {
+    "id": "mon_lemure",
+    "type": "MONSTER",
+    "name": "lemure",
+    "description": "A lemure resembles a molten mass of flesh with a vaguely humanoid head and torso.",
+    "default_faction": "magical_beast",
+    "bodytype": "blob",
+    "species": [ "MAGICAL_BEAST" ],
+    "diff": 2,
+    "volume": "62500 ml",
+    "weight": 45000,
+    "hp": 20,
+    "speed": 50,
+    "material": [ "flesh" ],
+    "symbol": "0",
+    "color": "magenta",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 4,
+    "melee_dice": 1,
+    "melee_dice_sides": 4,
+    "melee_cut": 1,
+    "dodge": 2,
+    "vision_day": 40,
+    "vision_night": 40,
+    "harvest": "exempt",
+    "anger_triggers": [ "PLAYER_CLOSE" ],
+    "death_function": [ "MELT" ],
+    "flags": [ "SEES", "SMELLS", "HEARS", "STUMBLES", "PATH_AVOID_FIRE" ]
```

Comment:
No regeneration factor for lemure?
![image](https://user-images.githubusercontent.com/51099123/66353575-55e0ab00-e930-11e9-92a4-55bd196023f9.png)
https://forgottenrealms.fandom.com/wiki/Lemure

Reply:
Why are you so evil?

Reply:
😈

Reply:
I will add Regenerate 1 to it, but revival ability is out of question at current time.
Mainly from lack of holy element, weapons or water.

Reply:
i don't have a problem with giving it the revives flag. i don't want to add an extra element to check whether or not the revival is possible, so butchery or pulping should be fine to work close enough

Reply:
Ok. I will add revives flag and change its death function into normal one, then.